jenkins/Jenkinsfile

```diff
@@ -159,6 +159,7 @@ pipeline {
             description: "The tag of the welder image to pull in docker image build. Used in the prepare script. Please note this default is possibly an old hash.")
         string(name: "openidc_proxy", defaultValue: "2.3.1_2",
             description:"The tag of the openidc proxy image to pull in docker image build. Used in the prepare script.")
+        string(name: "branch", defaultValue: "develop")
```

Comment:
Ah thanks for adding this - I've always manually changed the branch when I want to test something.